.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: crystal
+
+crystal:
+  - latest
+  - nightly
```

Comment:
Mind providing the reasoning for also testing on nightly? It seems like it could misreport a build as failing if breaking changes are introduced in nightly.

Reply:
If the CI job fails on `nightly` (e.g. because some Crystal API has been removed / changed behavior), there might be a solution to support both `latest` _and_ `nightly` (e.g. by using some other future-proof API). If the CI job fails on `nightly` because the whole `nightly` build env is broken (when Crystal devs have screwed up), the failure could simply be ignored until a fixed `nightly` is available. I assume testing on `nightly` makes it easier to support upcoming Crystal releases and improve overall code quality. I'm new to Crystal though and don't know how frequently breaking changes are introduced to the language.

Reply:
You may consider using `crystal build src/invidious.cr --error-on-warnings` in that case instead of two builds on `latest` and `nightly`.

Otherwise looks good. If you'd like to mark as resolved I'll consider this ready to merge :+1:.

Reply:
Building with `--error-on-warnings` now and removed `nightly`.